### PR TITLE
Saved addresses - context menu

### PR DESCRIPTION
**Pull request by saledjenic**
This pr introduces actions for the saved address item's context menu.

Apart from that, it makes adding/editing popup's content scrollable, reactors QR popup to be reusable and introduces a new UI design for it.

Changes are split into several commits for easier review.

### CODE REVIEW

**Review by alaibe — APPROVED** [review]

File: src/app/modules/main/wallet_section/saved_addresses/io_interface.nim [hunk 1]
```diff
@@ -21,6 +21,9 @@ method createOrUpdateSavedAddress*(self: AccessInterface, name: string, address:
   chainShortNames: string) {.base.} =
   raise newException(ValueError, "No implementation available")
 
+method updatePrefferdChains*(self: AccessInterface, address: string, chainShortNames: string) {.base.} =
```
Comment (Khushboo-dev-cpp) [inline src/app/modules/main/wallet_section/saved_addresses/io_interface.nim:24]: preferred?

File: src/app/modules/main/wallet_section/saved_addresses/module.nim [hunk 2]
```diff
@@ -61,6 +61,12 @@ method createOrUpdateSavedAddress*(self: Module, name: string, address: string,
   chainShortNames: string) =
   self.controller.createOrUpdateSavedAddress(name, address, ens, colorId, chainShortNames)
 
+method updatePrefferdChains*(self: Module, address: string, chainShortNames: string) =
```
Comment (Khushboo-dev-cpp) [inline src/app/modules/main/wallet_section/saved_addresses/module.nim:64]: here as well

File: src/app/modules/main/wallet_section/saved_addresses/view.nim [hunk 3]
```diff
@@ -46,6 +46,9 @@ QtObject:
     chainShortNames: string) {.slot.} =
     self.delegate.createOrUpdateSavedAddress(name, address, ens, colorId, chainShortNames)
 
+  proc updatePrefferdChains*(self: View, address: string, chainShortNames: string) {.slot.} =
```
Comment (Khushboo-dev-cpp) [inline src/app/modules/main/wallet_section/saved_addresses/view.nim:49]: here as well

File: ui/StatusQ/src/StatusQ/Controls/StatusRoundButton.qml [hunk 4]
```diff
@@ -134,7 +134,7 @@ Rectangle {
     QtObject {
         id: d
         readonly property color iconColor: !statusRoundButton.enabled ? statusRoundButton.icon.disabledColor :
-                                                                        (statusRoundButton.enabled && statusRoundButton.hovered) ? statusRoundButton.icon.hoverColor :
+                                                                        (statusRoundButton.enabled && (statusRoundButton.hovered || statusRoundButton.highlighted)) ? statusRoundButton.icon.hoverColor :
```
Comment (Khushboo-dev-cpp) [inline ui/StatusQ/src/StatusQ/Controls/StatusRoundButton.qml:137]: wondering if this has any impact on places this button was already used?

File: ui/app/AppLayouts/Wallet/stores/RootStore.qml [hunk 5]
```diff
@@ -352,6 +352,10 @@ QtObject {
         walletSectionSavedAddresses.createOrUpdateSavedAddress(name, address, ens, colorId, chainShortNames)
     }
 
+    function updatePrefferdChains(address, chainShortNames) {
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Wallet/stores/RootStore.qml:355]: here as well

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM but it would be great if you could add a video of the working feature in the PR. Also need to rename that one function.

**Comment by saledjenic** [comment]
@Khushboo-dev-cpp method name updated, the video is coming. Thanks.

**Comment by saledjenic** [comment]
@Khushboo-dev-cpp here is the video of 4 context menu options in action:


https://github.com/status-im/status-desktop/assets/86303051/1c857289-d47f-4557-92eb-d536c115306e

**Pull request merged**